DRAFT.md

```diff
@@ -842,25 +846,30 @@ constexpr indirect& operator=(const indirect& other);
   if any, is destroyed using `allocator_traits<allocator_type>::destroy` and then
   deallocated using `allocator_traits<allocator_type>::deallocate`.
 
-  Otherwise, if `alloc == other.alloc` and `this` is not valueless, equivalent
-  to `T tmp(*other); std::swap(tmp, **this)`.
+  Otherwise, if `alloc == other.alloc` and `this` is not valueless, the owned
+  object is assigned to `*other`.
 
-  Otherwise , if `alloc != other.alloc` or `this` is valueless, a new owned
+  Otherwise, if `alloc != other.alloc` or `this` is valueless, a new owned
   object is constructed in `this` using
   `allocator_traits<allocator_type>::construct` with the owned object from
   `other` as the argument, with memory allocated using either the allocator in
   `this` or the allocator in `other` if the allocator needs updating. The
-  previous owned object in this, if any, is destroyed using
+  previously owned object in this, if any, is destroyed using
   `allocator_traits<allocator_type>::destroy` and then deallocated using
   `allocator_traits<allocator_type>::deallocate`.
 
   If the allocator needs updating, the allocator in `this` is replaced with a
   copy of the allocator in `other`.
 
-  No effects if an exception is thrown.
-
 3. _Returns_: A reference to `*this`.
 
```

Comment:
The following sequence of numbers needs to be updated following  the addition if this statement

Reply:
Well-spotted! Fixed.